src/index.ts

```diff
@@ -176,3 +179,30 @@ async function statusOk(res: Response, logger: Logger): Promise<boolean> {
     logger.debug('testing response for whether it is "ok". has status: ', res.status, ' debug: ', JSON.stringify(res))
     return String(res.status)[0] === '2'
 }
+
+function getProperties(event: PluginEvent, { config }: SalesforcePluginMeta): Properties {
+    // Spreading so the TypeScript compiler understands that in the
+    // reducer there's no way the properties will be undefined
+    const { properties } = event
+
+    if (!properties) {
+        return {}
+    }
+
+    if (!config.propertiesToInclude?.trim()) {
+        return properties
+    }
+
+    const allParameters = config.propertiesToInclude.split(',')
+    const propertyKeys = Object.keys(properties)
+
+    const availableParameters = allParameters.reduce<Record<string, any>>((acc, currentValue) => {
+        if (propertyKeys.includes(currentValue.trim())) {
+            acc[currentValue] = properties[currentValue]
```

Comment:
On one line `currentValue.trim()` is used, but on the other it's just `currentValue`. Could use something like `const trimmedPropertyKey = currentValue.trim()`